packages/camera/camera_web/pubspec.yaml

```diff
@@ -2,7 +2,7 @@ name: camera_web
 description: A Flutter plugin for getting information about and controlling the camera on Web.
 repository: https://github.com/flutter/packages/tree/main/packages/camera/camera_web
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+camera%22
-version: 0.3.2+4
+version: 0.4.0
```

Comment:
(I suspect this needs to be `version: 0.3.3`)

Reply:
changed to 0.3.3